tests/tracee_test.go

```diff
@@ -126,3 +129,92 @@ func TestWebhookIntegration(t *testing.T) {
 	t.Log("Terminating the Tracee container...")
 	assert.NoError(t, exec.Command("docker", "kill", containerID).Run())
 }
+
+type traceeContainer struct {
+	testcontainers.Container
+}
+
+func setupTraceeContainer(ctx context.Context, tempDir string) (*traceeContainer, error) {
+	req := testcontainers.ContainerRequest{
+		Image:      "tracee",
+		Privileged: true,
+		BindMounts: map[string]string{tempDir: tempDir},
+		Name:       "tracee",
+		AutoRemove: true,
+	}
+	container, err := testcontainers.GenericContainer(ctx, testcontainers.GenericContainerRequest{
+		ContainerRequest: req,
+		Started:          true,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	return &traceeContainer{Container: container}, nil
+}
+
+func setupTraceeTrainerContainer(ctx context.Context, sigid string) (*traceeContainer, error) {
+	req := testcontainers.ContainerRequest{
+		Image:      "tracee-trainer",
+		Entrypoint: []string{"/runner", sigid},
+		Privileged: true,
+		Name:       "tracee-trainer",
+		AutoRemove: true,
+	}
+	container, err := testcontainers.GenericContainer(ctx, testcontainers.GenericContainerRequest{
+		ContainerRequest: req,
+		Started:          true,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	return &traceeContainer{Container: container}, nil
+}
+
+func TestTraceeSignatures(t *testing.T) {
+	tempDir := "/tmp/tracee"
+	err := os.MkdirAll(tempDir, os.ModePerm) // TODO: Investiagte why passing ioutil.Temp does not work with Tracee mount
```

Comment:
what does this comment mean? the test doesn't work?

Reply:
This test works as is. When it comes to creating temporary dirs, ideally having a call to `ioutil.TempDir` should do it. But I think there's some inherent assumption within tracee (could potentially be a bug) that assumes that directory to be `/tmp/tracee`. Passing in a temp dir created with ioutil.TempDir didn't work as expected.

Need to further investigate. I can take another look.